app/Traits/Services/HasWebhookPayload.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Traits\Services;
+
+trait HasWebhookPayload
+{
+    public function getPayload(): array
+    {
+        if (method_exists($this, '__serialize')) {
+            return $this->__serialize();
+        }
+
+        return [];
```

Comment:
Is it possible for an Event to not be serializable?

Reply:
I kinda tried to use SerializesModels here, there has to be a better way

Reply:
Basically - if the Event uses `SerializesModels`, use that, otherwise, or if necessary, we could modify the payload per Event just overriding the `getPayload` method